locales/pt-BR/translation.json

```
@@ -1,7 +1,7 @@
 {
     "$Language": {
         "en": "English",
-        "pt-BR": "Português (Brasil)"
+        "pt-BR": "(Português (Brasil"
```

Hahahahahahah Does it really work? God, I have no idea what's happening there.
I liked the idea of the workaround, but I think we should go with a more "Generic" solution:
```suggestion
        "pt-BR": "Português - Brasil"
```

Please note that you should also change the string on `en/translation.json`.
Would you please do that?

---

Will do, I didn't change the english translation one because it rendered correctly already, but if you want to have a "-" instead I will update both :)

---

in addition, for consistency, should English translate to "Inglês" when the language is set to Portugese, or do you want to keep it listed as "English"? @thamara

---

No need to "translate" English. We think it's better to keep the language name as their original name, no matter in which locale

---

Ok, I pushed a new commit, do I need to do anything else for it to be visible to you? ^^ Sorry, it still confuses me a bit to PR repos that aren't mine

---

No need. All good :)